### PR TITLE
Adds a new non-selectable trait, smelly, which assistants start with.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus
add: assistants now start with the smelly trait which makes them smellier 50% faster.
fix: fixes some null values for some of the hygiene adjustments
/:cl:

ok so like, assistants are sacks of shit so to showcase this they become smellier 50% faster

### CODE REVIEW

**Comment by imsxz** [comment]
assistants die

**Comment by optimumtact** [comment]
funny but this is just silly

**Comment by Qustinnus** [comment]
they will die of attrition in their cesspits. i should make this trait worsen if youre around more people with the trait because its worse in cesspits.

**Comment by Shadowflame909** [comment]
Quite Pungent My Dear

This is 4chan isn't it

**Comment by subject217** [comment]
24 HOUR RULE

**Comment by Qustinnus** [comment]
i feel like its an acceptable level of silly, what if I put it behind a prob(20) or something

**Comment by optimumtact** [comment]
it's not

**Comment by Qustinnus** [comment]
sad

**Comment by subject217** [comment]
this is the nerf assistant mains needed, RIP

File: code/__DEFINES/traits.dm [hunk 1]
```diff
@@ -65,6 +65,7 @@
 #define TRAIT_ABDUCTOR_TRAINING "abductor-training"
 #define TRAIT_ABDUCTOR_SCIENTIST_TRAINING "abductor-scientist-training"
 #define TRAIT_SURGEON           "surgeon"
+#define TRAIT_SMELLY			"nervere"
```
Comment (Qustinnus) [inline code/__DEFINES/traits.dm]: also why wouldnt you want this in your codebase

**Comment by PKPenguin321** [comment]
#NotAllAssistants

**Comment by Qustinnus** [comment]
ugh i just realized this PR fixes bugs but im too lazy to make a new pr. the pain.

**Comment by crossedfall** [comment]
#FreeTheStink #LetThemSmell #Ass-istants

**Pull request closed without merging**